kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kms/aws/AwsKmsLocal.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.installation.kms.aws;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.net.URI;
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.fabric8.kubernetes.api.model.ServicePort;
+
+import io.kroxylicious.kms.provider.aws.kms.AwsKmsTestKmsFacade;
+import io.kroxylicious.systemtests.Environment;
+import io.kroxylicious.systemtests.k8s.exception.KubeClusterException;
+import io.kroxylicious.systemtests.resources.manager.ResourceManager;
+import io.kroxylicious.systemtests.utils.DeploymentUtils;
+import io.kroxylicious.systemtests.utils.KafkaUtils;
+import io.kroxylicious.systemtests.utils.NamespaceUtils;
+import io.kroxylicious.systemtests.utils.TestUtils;
+
+import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
+
+/**
+ * The type Aws kms local.
+ */
+public class AwsKmsLocal implements AwsKmsClient {
+    public static final String LOCALSTACK_SERVICE_NAME = "localstack";
+    public static final String LOCALSTACK_DEFAULT_NAMESPACE = "localstack";
+    public static final String LOCALSTACK_HELM_REPOSITORY_URL = "https://localstack.github.io/helm-charts";
+    public static final String LOCALSTACK_HELM_REPOSITORY_NAME = "localstack";
+    public static final String LOCALSTACK_HELM_CHART_NAME = "localstack/localstack";
+    private static final Logger LOGGER = LoggerFactory.getLogger(AwsKmsLocal.class);
+    private static final String AWS_LOCAL_CMD = "awslocal";
```

Comment:
ditto, document the dependency in the appropriate part of the DEV_GUIDE.

Reply:
we don't need nothing extra for `awslocal`, this is part of the `localstack` helm chart installation.